flycheck.el

```diff
@@ -3850,17 +3874,26 @@ overlays."
             (flycheck-error-level-severity (flycheck-error-level err)))
       t)))
 
-(defun flycheck-next-error-pos (n &optional reset)
-  "Get the position of the N-th next error.
+(defun flycheck-next-error-overlay-pos (n &optional reset)
+  "Get the position of the N-th next error overlay in buffer.
```

Comment:
I renamed this function because I found the name misleading, since after #1427 overlays may now point to errors in other files, and so the position of an overlay and the corresponding error need no longer be the same. However, there may be other places where this new distinction between overlays and errors also applies, and I didn't fix any other occurrences. So, maybe it's better to not change the name here?